cypress.config.ts

```diff
@@ -19,8 +19,7 @@ const getSpecPattern = (): string[] => {
 
   return [
     ...activePaths,
-    'cypress/e2e/tests/pages/**/*.spec.ts',
-    'cypress/e2e/tests/navigation/**/*.spec.ts'
+    'cypress/e2e/tests/**/*.spec.ts',
```

Comment:
Could you please revert this part?
We are excluding the setup as is optionally included in the `activePaths`.